backend/src/GraphQl/Institutions/InstitutionMutations.cs

```diff
@@ -182,7 +182,7 @@ CancellationToken cancellationToken
                 userManager
             ).ConfigureAwait(false)
            )
-            return new VerifyInstitutionPayload(
+            return new VerifyInstitutionPayload(    
```

Comment:
Please remove the spaces you added here after the last character with meaning, namely `(`.